src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,7 @@
 const booleanToWord = (boolean) => {
-
-}
+     if (boolean) {
+        return 'Yes'
+    } else { return 'No'}
```

Comment:
You can use the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here:

```js
return boolean ? 'Yes' : 'No'
```